action/action-simulator/src/main/java/com/powsybl/action/simulator/loadflow/LoadFlowActionSimulator.java

```diff
@@ -401,4 +408,56 @@ private LoadFlowResult runTest(RunningContext context, Network networkForTry, Ac
             throw new PowsyblException(e);
         }
     }
+
+    private interface NetworkCopyStrategy {
+
+        void clearState();
+
+        Network restoreState(String stateId);
+
+        void saveState();
+    }
+
+    private class DeepCopyStrategy implements NetworkCopyStrategy {
+        private byte[] contextNetwork;
+
+        @Override
+        public void clearState() {
+            // Nothing to do
```

Comment:
I have a doubt here: I think it could be useful to release `contextNetwork` memory by setting this field to null. Trying this, unit tests produce a NullPointerException. I think something is missing: we should be able to free the memory when the `NetworkCopyStrategy` instance if not used.